### ISSUE TITLE
En dash is indistinguishable from a minus

### ISSUE THREAD

**Issue by dyavc** [abrudz/APL386#9]
This is based on a user's report about APL385 Unicode, so I thought I'd try APL386

For you to consider please,
⎕ucs 8211 45 ⍝ en dash then minus sign
⎕ucs 8212 ⍝ em dash

The user also says that em dash is not long enough.

**Comment by abrudz** [comment]
I agree that ndash could be slightly longer but note that this is a monospace font. Making mdash any longer would make it into a contiguous line. Here are 4 adjacent characters of each type:
![image](https://github.com/abrudz/APL386/assets/10961427/cfbb6501-cf0c-432b-b5b8-30cc56730cb7)
